xml/System.Diagnostics.Tracing/DiagnosticCounter.xml

```diff
@@ -137,9 +145,9 @@ After being disposed, this counter will do nothing, and its resources will be re
         <ReturnType>System.Diagnostics.Tracing.EventSource</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The EventSource that this counter is attached to.</summary>
         <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <remarks></remarks>
```

Comment:
The "To be added." message needs to stay untouched if the item won't be documented. It's considered the default message. Can you please put it back?

Reply:
Also, remarks are optional. Everything else is mandatory.